src/Components/Header/Header.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import youtube from '../../Assets/youtube.svg';
+import "./Header.css"
+import MenuSharpIcon from '@mui/icons-material/MenuSharp';
+import SearchSharpIcon from '@mui/icons-material/SearchSharp';
+import { useAuth } from '../../Contexts/Auth.context';
+import { useNavigate } from 'react-router-dom';
+export function Header() {
+  const {state,dispatch} = useAuth()
+  const navigate = useNavigate()
+  function handleNavigate()
+  {
+    if(!state.user){
+      navigate("/auth/login")
+    }
+    else
+    {
+      dispatch({type:"SIGN_OUT"})
+    }
+  }
```

Comment:
Suggestion : You can use ternary here.

Reply:
Thank you for the suggestion. Implemented